pkg/utils/display/display.go

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2023. Nydus Developers. All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package display
+
+import "fmt"
+
+func ByteToReadableIEC(b uint32) string {
```

Comment:
International Electrotechnical Commission. A [standard](https://en.wikipedia.org/wiki/Byte) for converting units.

For example, when converting `KiB` to `B`, it should be divided by 1024 instead of 1000.